ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml

```diff
@@ -612,6 +612,7 @@ images:
   to: clusterimageset-updater
 - dockerfile_literal: |-
     FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
+    RUN microdnf install -y tar gzip
```

Comment:
Why this is needed? I see that the job is failing but I am falling to find where exactly the job is running tar commands.....

Reply:
Ah, good to know. I also see that the only other infra-periodic that uses `tar` that way is `pr-reminder`. Perhaps we have found them all already.